src/commons/Reporter.ts

```diff
@@ -45,18 +45,72 @@ export namespace Reporter {
   let isStepClosed: boolean = true;
   let currentStepTitle: string;
   let customCommand: CustomCommand;
+  const networkActivity: { url: string; status: string; headers: object }[] = [];
+
+  /**
+   * Enable network audits for test run.
+   *
+   * Require adding devtools as a service in wdio.conf.js
+   * See https://webdriver.io/docs/devtools-service.html
+   *
+   * Will log all network logs to networkActivity that can be added to html report
+   * For more readable logs, we only log <url>, <status> and <headers> instead of whole request data
+   *
+   * Since devtools typing are missing ts ignore required in some cases such as browser.cdp(...)
+   *
+   * Example of usage:
+   * In beforeTest hook:
+   *      Reporter.enableNetworkAudits()
+   * In afterTest hook:
+   *      Reporter.addAttachment('Network Logs', { https: networkActivity }, 'application/json');
+   *    already integrated in Reporter.closeStep method in case of test failure
+   */
+  export function enableNetworkAudits(): void {
+    if (browser.capabilities.browserName === 'chrome') {
+      // @ts-ignore
+      browser.cdp('Network', 'enable');
```

Comment:
pls add jdoc, what kind of audit we taken

Reply:
added "   * Will log xhr and fetch responses
"